### PR TITLE
added simple build script to build CPAN modules on Debian unstable/sid

### PR DESCRIPTION
**Pull request by sbellon**
Very simple and without any handling of edge-cases, but may work as a starting point for others.

### CODE REVIEW

**Comment by mherger** [comment]
Do you believe I should re-build those libraries to include Debian based versions? This happens all the time, as Fedora would bump the Perl version first, but uses a more recent libc...

BTW: I usually use this https://github.com/LMS-Community/slimserver-vendor/blob/public/8.4/CPAN/Docker/Dockerfile.debian to build on Debian.

**Comment by michaelherger** [comment]
Would the following build work on your installation?

[CPAN-debian-unstable.zip](https://github.com/LMS-Community/slimserver-vendor/files/14741151/CPAN-debian-unstable.zip)

**Comment by sbellon** [comment]
Sorry for the late reply...

> Would the following build work on your installation?
> 
> [CPAN-debian-unstable.zip](https://github.com/LMS-Community/slimserver-vendor/files/14741151/CPAN-debian-unstable.zip)

Yes, that works straight out of the box. :-D

The only thing I noticed is, that in my LMS installation the modules are inside `x86_64-linux-thread-multi/` and when building the modules (and also in your zip file) they are inside `x86_64-linux-gnu-thread-multi/`. But when copying them over the supplied ones (or self-built ones), everything is just fine and works. :+1:

**Comment by michaelherger** [comment]
Thanks, added those file to LMS 9. If you could please give it a try with tomorrow's build? I could back port it to 8.5 after some successful testing on 9.

**Comment by sbellon** [comment]
Is there anything special I have to do when upgrading to LMS 9 ... or even more so, when then deciding to downgrade to 8.5 again at some point?

**Comment by michaelherger** [comment]
I don't think there's anything particular right now. But as soon as the great renaming begins, and folder names change etc. There will be no easy way back. We will try to migrate forward, but not backwards.

**Comment by sbellon** [comment]
I just upgraded to the latest 9.0.0 nightly from 8.5.0 release without problems. It started out of the box just fine.

Downgrading back to 8.5.0 worked - I just had to copy over the other modules and patch Audio/Scan.pm again.

TL;DR: your 9.0.0 build works perfectly find and I would love to see that included in 8.5 as well. ;-)

**Comment by michaelherger** [comment]
Thanks! I back ported to 8.5 and hope to release 8.5.1 soon.

**Comment by sbellon** [comment]
Should I somehow decline this PR then?

**Comment by michaelherger** [comment]
Closing advice: use Docker to build the binaries 😁. Thanks for the testing!

**Comment by sbellon** [comment]
> [...] use Docker to build the binaries 😁.

Definitely "yes" when building to redistribute them! In my setup I built them on the machine where LMS itself is running, so no need for Docker in that case. But I completely agree in general and am more than happy that from now on I do not have to do this manually any more. :-)
